requirements.txt

```diff
@@ -7,7 +7,8 @@ pytest-shard==0.1.2
 
 # Django deps:
 psycopg2-binary
-Django==4.2.7
+Django==4.2.7; python_version < '3.10'
+Django==5.0.0; python_version >= '3.10'
```

Comment:
Not sure if this can cause us problems down the road. I'm sure we will be able to solve them in the future as they pop up.

If we need to write Django 5.0-specific tests, we can specifically add `skip: sys.version_info < (3, 10)` to them.